app/views/users/show.html.haml

```diff
@@ -1,9 +1,14 @@
-%br/
+#profile
+  %h2
+    User: #{@user.username}
+  - if user_signed_in? && current_user.returning_instructor? && current_user.admin?
```

Comment:
The email should show if either:
* The current user is the same one whose profile is being shown (as in, current_user == @user )
* The current user is an admin